numba/tests/test_function.py

```diff
@@ -0,0 +1,575 @@
+import types as pytypes
+from numba import njit, function, cfunc, types, int64, int32
```

Comment:
The features in this PR depends on importing `function`. It should be auto-initialized. It can be done like: https://github.com/numba/numba/blob/ef119bcd1733ff49d71bdf2da8a66e91bb704f83/numba/targets/cpu.py#L65

Reply:
Fixed.